CRISPResso2/CRISPRessoPooledCORE.py

```diff
@@ -647,12 +649,60 @@ def main():
 
             N_READS_ALIGNED=get_n_aligned_bam(bam_filename_amplicons)
 
-            s1=r"samtools view -F 4 %s 2>>%s | grep -v ^'@'" % (bam_filename_amplicons,log_filename)
-            s2=r'''|awk '{ gzip_filename=sprintf("gzip >> OUTPUTPATH%s.fastq.gz",$3);\
-            print "@"$1"\n"$10"\n+\n"$11  | gzip_filename;}' '''
+            if args.amplicons_file != '':
+                bam_iter = CRISPRessoShared.get_command_output(
+                    'samtools sort {bam_file} | samtools view -F 4 2>> {log_file}'.format(
+                        bam_file=bam_filename_amplicons,
+                        log_file=log_filename,
+                    ),
+                )
+                curr_file, curr_chr, curr_pos = None, None, None
```

Comment:
can get rid of curr_pos because we're not tracking it -- assign all things that are aligned to curr_chr to that file.